guides/doc-Provisioning_Guide/topics/proc_implementing-pxe-less-discovery.adoc

```diff
@@ -0,0 +1,46 @@
+[[implementing-pxe-less-discovery]]
+= Implementing PXE-less Discovery
+
+{Project} provides a PXE-less Discovery service that operates without the need for PXE-based services (DHCP and TFTP). You accomplish this using {ProjectServer}'s Discovery image. Once a discovered node is scheduled for installation, it uses `kexec` command to reload Linux kernel with OS installer without rebooting the node.
+
+ifeval::["{build}" == "satellite"]
+[NOTE]
+Discovery `kexec` is a Technology Preview feature.
+endif::[]
+
+image::PXEless-mode.png[]
+
+If you have not yet installed the Discovery service or image, see xref:building-a-discovery-image[].
+
+The ISO for the Discovery service resides at `/usr/share/foreman-discovery-image/` and is installed using the `foreman-discovery-image` package.
+
+.Attended Use
+
+This ISO acts as bootable media. Copy this media to either a CD, DVD, or a USB stick. For example, to copy to a USB stick at `/dev/sdb`:
+
+[options="nowrap" subs="+quotes"]
+----
+# dd bs=4M \
+if=/usr/share/foreman-discovery-image/foreman-discovery-image-3.4.4-5.iso \
+of=/dev/sdb
+----
```

Comment:
This could be a module perhaps?

Reply:
I added it as a step to the overview. I don't know why it was on its own like that.